### ISSUE TITLE
Need to update the formControl docs to read disable instead of disabled

### ISSUE THREAD

**Issue by ejzn** [blackberry/WebWorks-API-Docs#429]
As per above.

**Comment by bryanhiggins** [comment]
@kwallis - web security & child browser use "disabled" rather than "disable". I propose we accept either for formControl to make it consistent but not break existing apps

https://github.com/blackberry/BB10-WebWorks-Framework/issues/391

**Comment by ejzn** [comment]
Doesn't child browser use disable as well?

 if (childBrowser) {
                    widgetConfig.enableChildWebView = ((childBrowser + '').toLowerCase() === 'disable') === false;
                }

**Comment by bryanhiggins** [comment]
Yep - same for web security. The microsite docs are wrong for all three.

**Comment by ejzn** [comment]
Exciting. My vote is to change the docs.

**Comment by bryanhiggins** [comment]
I added an issue to the microsite for this to be changed
